gourmet/tests/test_ica_se_plugin.py

```diff
@@ -26,7 +26,7 @@ def _read_html(self):
         filename = os.path.join(os.path.dirname(__file__),
                                 'recipe_files',
                                 (os.path.splitext(os.path.basename(__file__))[0])[5:-7]+".html")
-        return (open(filename).read())
+        return (open(filename, encoding="utf8").read())
```

Comment:
I'd rather have a context manager here: whilst in CPython, the file will be closed when the reference count drops to zero, it's not guaranteed by all implementations (and not necessarily guaranteed for future CPython). It's not mega important, but I would find it better more tasteful to use context managment.

Reply:
Yep, when you leave an open file descriptor, a kitten dies somewhere... But I decided not to comment on that because it refers to another ticket (#26). We can fix it there.

Reply:
It is easy to switch to the `with open()` version here, but I just changed the stuff needed to at least run the test.